.gitignore

```diff
@@ -1,5 +1,3 @@
 
 node_modules
 
-build
```

Comment:
@Sajj96 we don't need the build folder in source control. It should be created at the point of deployment